modules/CmpdReg/src/client/custom/LotView_Custom.inc

```diff
@@ -62,7 +62,7 @@
 			<div>
 				<label class="FormLabel">Lot Number:</label>
 				<input type="textfield" class="FormInput lotNumber" <%- (saved | isVirtual | !allowManualLotNumber) ? 'disabled=true': '' %> value="<%- lotNumber %>"/>
-				<img class="insertNextAutoLotNumberButton" href="#" title="Fill with next lot number" src="/CmpdReg/client/images/edit.png" <%- (saved | isVirtual | !allowManualLotNumber) ? 'style="display:none;"': 'style="width: 15px;"' %>/>
+				<img class="insertNextAutoLotNumberButton" href="#" title="Fill with next lot number" src="/CmpdReg/client/images/edit.png" <%= (saved | isVirtual | !allowManualLotNumber) ? 'style="display:none;"': 'style="width: 15px;"' %>/>
```

Comment:
This was broken since we added this escape pattern which does not eval the code.